docs/dev/operators.md

```diff
@@ -14,7 +14,7 @@ LABEL io.openshift.release.operator=true
 Ensure your image is published into the cluster release tag by ci-operator
 Wait for a new release payload to be created (usually once you push to master in your operator).
```

Comment:
nit: will be nice to add a period - `tag by ci-operator. Wait for a new release...`